### PR TITLE
feat(containers): deployment command interface

### LINKED ISSUE #620: containers: add support for interface "io.edgehog.devicemanager.apps.DeploymentCommand"
ref: #614 
interface: https://github.com/edgehog-device-manager/edgehog-astarte-interfaces/pull/64/files#diff-93a9056322cb77898591d9eb9a23264310c6510575165be004dc206fbb5a5644

### PR DESCRIPTION
**Pull request by noaccOS**
allows sending a command to a release

closes #620

### CODE REVIEW

**Comment by noaccOS** [comment]
_I hate credo_

File: backend/lib/edgehog/devices/device/device.ex [hunk 1]
```diff
@@ -227,6 +227,19 @@ defmodule Edgehog.Devices.Device do
       manual ManualActions.SetLedBehavior
     end
 
+    update :send_application_command do
```
Comment (rbino) [inline backend/lib/edgehog/devices/device/device.ex:230]: Why the inconsistent naming application/deployment?

File: backend/lib/edgehog/devices/devices.ex [hunk 2]
```diff
@@ -69,6 +69,7 @@ defmodule Edgehog.Devices do
       update Device, :add_device_tags, :add_tags
       update Device, :remove_device_tags, :remove_tags
       update Device, :set_device_led_behavior, :set_led_behavior
+      update Device, :send_device_application_command, :send_application_command
       create HardwareType, :create_hardware_type, :create
```
Comment (lusergit) [inline backend/lib/edgehog/devices/devices.ex:73]: I think this is not needed. We should not directly expose the  `send_application_command` action trough graphql

File: backend/lib/edgehog/devices/device/device.ex (hunk 1 above)
Comment (noaccOS) [reply backend/lib/edgehog/devices/device/device.ex:230]: In my mind we talk to the device with the deployment but the api exposes applications/releases instead

_Should have probably been "release_command" actually_

**Review by lusergit — APPROVED** [review]

File: backend/lib/edgehog/devices/device/device.ex (hunk 1 above)
Comment (noaccOS) [reply backend/lib/edgehog/devices/device/device.ex]: Changed it to `:send_release_command`

**Review by rbino — APPROVED** [review]

**Pull request merged**